proxy/http/HttpConfig.cc

```diff
@@ -1394,8 +1413,8 @@ HttpConfig::reconfigure()
 
   // Local Manager
   params->synthetic_port = m_master.synthetic_port;
-
-  m_id = configProcessor.set(m_id, params);
+  params->codeNegCache   = m_master.codeNegCache;
```

Comment:
This was unclear to me too, but amc told me that the intent was to make this configurable (but it obviously isn't in this patch). So ignore my comment on the switch statement, it would not suffice if you plan on making this vector / array configurable.